.github/workflows/inspektor-gadget.yml

```diff
@@ -331,16 +331,64 @@ jobs:
           -czf ig-linux-${{ matrix.platform }}.tar.gz -C ${{ matrix.platform }} \
           ig LICENSE
         rm -rf ${{ matrix.platform }}
-    - name: CWE checks for ig-linux-${{ matrix.platform }}
-      uses: github/codeql-action/analyze@v3
-      with:
-        category: ig-linux-${{ matrix.platform }}
     - name: Add ig-linux-${{ matrix.platform }}.tar.gz as artifact.
       uses: actions/upload-artifact@v3
       with:
         name: ig-linux-${{ matrix.platform }}-tar-gz
         path: /home/runner/work/inspektor-gadget/inspektor-gadget/ig-linux-${{ matrix.platform }}.tar.gz
 
+  build-ig-cwe:
+    name: Check ig CWE
+    # level: 1
+    needs:
+      - btfgen
```

Comment:
I think we can omit btfgen altogether on this new job. It only generates some BTF files that are embed in the binary and it shouldn't produce any CWE alert.